### PR TITLE
Consistency fixes

### PR DESCRIPTION
**Pull request by rbuzatu90**

### CODE REVIEW

File: jobs/run_collect.sh [hunk 1]
```diff
@@ -32,11 +32,10 @@ run_wsmancmd_with_retry $hyperv01 $WIN_USER $WIN_PASS 'powershell -executionpoli
 run_wsmancmd_with_retry $hyperv01 $WIN_USER $WIN_PASS 'sc qc nova-compute >> \\'$FLOATING_IP'\openstack\logs\'${hyperv01%%[.]*}'\nova_compute_service.log'
 run_wsmancmd_with_retry $hyperv01 $WIN_USER $WIN_PASS 'sc qc neutron-hyperv-agent >> \\'$FLOATING_IP'\openstack\logs\'${hyperv01%%[.]*}'\neutron_hyperv_agent_service.log'
 
+run_wsmancmd_with_retry $hyperv02 $WIN_USER $WIN_PASS 'powershell -ExecutionPolicy RemoteSigned Copy-Item -Recurse C:\OpenStack\Log\* \\'$FLOATING_IP'\openstack\logs\'${hyperv02%%[.]*}'\'
```
Comment (ociuhandu) [inline jobs/run_collect.sh]: we should, in fact, collect all logs in the local folder (including eventlog etc) and only then copy the whole folder.

Comment (rbuzatu90) [inline jobs/run_collect.sh]: So this should be the last thing to do after we put the logs into files , right ?

Comment (ociuhandu) [inline jobs/run_collect.sh]: Yes, see lines 36/37.

Comment (rbuzatu90) [inline jobs/run_collect.sh]: So to close this PR, will put export-eventlog.ps1 to dump logs in C:\OpenStack\Log\ (openstack`s default log folder) and after that copy the whole dir to remote devstack location.
It is ok like so ?

Comment (ociuhandu) [inline jobs/run_collect.sh]: Yep, but I'd update the collection of eventlogs based on the sample provided: http://paste.ubuntu.com/15482703/

**Pull request merged**